docs/standard/serialization/system-text-json/snippets/how-to/csharp/Utf8ReaderPartialRead.cs

```diff
@@ -7,74 +7,85 @@ public class Utf8ReaderPartialRead
     {
         public static void Run()
         {
-            var jsonString = @"{
```

Comment:
Apropos, I'm wondering if it might be worth updating _all_ code snippets to use triple-quote literals -- the readability gains are so much worth it.